.github/workflows/main.yml

```diff
@@ -55,32 +56,44 @@ jobs:
       - name: install requirements
         run: pip install -r requirements.txt
       - name: download ubuntu results
-        uses: actions/download-artifact@v1
+        uses: actions/download-artifact@v2
         with:
           name: results-ubuntu-latest
+          path: /tmp/results-ubuntu
       - name: download macos results
-        uses: actions/download-artifact@v1
+        uses: actions/download-artifact@v2
         with:
           name: results-macos-latest
+          path: /tmp/results-macos
       - name: download windows results
-        uses: actions/download-artifact@v1
+        uses: actions/download-artifact@v2
         with:
           name: results-windows-latest
+          path: /tmp/results-windows
+      - name: configure aws credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-2
+      - name: dvc pull existing
+        run: dvc pull
       - name: join results
         run: |
-          mkdir results
-          cp -r results-macos-latest/* results/
-          cp -r results-windows-latest/* results/
-          cp -r results-ubuntu-latest/* results/
+          cp -r /tmp/results-windows/* results/
+          cp -r /tmp/results-macos/* results/
+          cp -r /tmp/results-ubuntu/* results/
```

Comment:
Are they on the same fs? If so, could you mv it so it is faster? Though I'm not even sure it takes much time to begin with.

Reply:
> Are they on the same fs?

Not sure, would need to debug.

They have few MB each, so I think there is no need to optimize them for now.

Reply:
Got it. Let's leave as is then. Thanks!